### DISCUSSION TITLE
Is there a way to record mono audio / set channels to one?

### DISCUSSION THREAD

**Discussion by ospaarmann** [llfbandit/record#174] [Q&A] [answered]
First of all thank you for the great package! I am trying to find a way to record **mono audio**. I get better results in transcriptions when the audio file only has **one channel**. I did not find anything in the documentation or the repo. I use record in a Flutter app for iOS and Android. Thanks!

```dart
// recording setup
await _record.start(
        path: filePath,
        encoder: AudioEncoder.aacLc, // by default
        bitRate: 128000, // by default
        samplingRate: 44100, // by default
      );
```

**Comment by llfbandit** [comment, marked as answer]
Yes you can with `numChannels` argument on `start` method.
I would recommend you to try the latest v5 beta with PCM streaming. Maybe it could unlock you live retranscription feature?

**Comment by ospaarmann** [comment]
That worked. Thanks! I already built a little convertor into my backend using [FFmpeg](https://ffmpeg.org/), but directly recording mono is better. It will save my users some bandwidth, and mixing two channels into one after the recording is challenging since any out-of-phase stereo will cancel out (unless properly handled).

Regarding the live transcription feature: I already built a prototype with [mic_stream](https://pub.dev/packages/mic_stream) with websockets, using my backend as a streaming proxy. But I wasn't happy with the results from the transcription API I'm using ([Deepgram](https://deepgram.com/)). The punctuation wasn't working as well as I expected. 

But I will definitely play with it and use it as soon as it makes sense!
